river/storage_formats.py

```diff
@@ -314,16 +359,14 @@ def _write_feather(df, tmpfile, *args, **kwargs):
 
 ##############################################################################
 
-# TODO
-#######
-# ORC #
-#######
-
-##############################################################################
-
 format_fn_map = {
    'avro': avro,
    'csv': csv,
+   'csv.gz': csv,
+   'csv.zip': csv,
```

Comment:
does reading a csv.zip using pandas automatically unzip it for you?

Reply:
Yes! I actually had written entire separate code to unzip the file before reading it in, and THEN of course is when I learned that pandas can automatically decompress when reading, and it can also write directly to a compressed file as well

Reply:
That said, I actually discovered that this will require a bit more work to get functioning properly - I'll let you know when it's ready for re-review!